redis/src/test/scala/zio/redis/StringsSpec.scala

```diff
@@ -1435,6 +1437,92 @@ trait StringsSpec extends BaseSpec {
             len <- strLen(key).either
           } yield assert(len)(isLeft(isSubtype[WrongType](anything)))
         }
+      ),
+      suite("getEx")(
+        testM("value exists after removing ttl") {
+          for {
+            key    <- uuid
+            value  <- uuid
+            _      <- pSetEx(key, 10.millis, value)
+            exists <- getEx(key, true)
+            _      <- ZIO.sleep(20.millis)
+            res    <- get(key)
+          } yield assert(res.isDefined)(equalTo(true)) && assert(exists)(equalTo(Some(value)))
+        } @@ eventually,
```

Comment:
Just a question why is this test `@eventually`?

Reply:
This is because sleep is unreliable and subject to CPU and thread scheduling. 20ms is likely to cause failure. Here, it is similar to CAS. (guess)

Reply:
Ah true, I forgot that we use the Live Clock in these tests.